components-api/src/main/scala/pl/touk/nussknacker/engine/api/component/DefaultAdditionalComponentsUIConfigProvider.scala

```diff
@@ -0,0 +1,7 @@
+package pl.touk.nussknacker.engine.api.component
+
+class DefaultAdditionalComponentsUIConfigProvider extends AdditionalComponentsUIConfigProvider {
+
+  override def getAllForCategory(category: String): Map[String, SingleComponentConfig] = Map.empty
```

Comment:
we could get rid of the `category` argument if the class instance is bound to one already, for example via `modelDataForType`, which is currently the only usage